lib/sidekiq/batch/status.rb

```diff
@@ -43,6 +54,27 @@ def child_count
         Sidekiq.redis { |r| r.hget("BID-#{bid}", 'children') }.to_i
       end
 
+      def success_pct
+        return 0 if total == 0
+        ((total - pending) / Float(total)) * 100
+      end
+
+      def pending_pct
+        return 0 if total == 0
+        ((pending - failures) / Float(total)) * 100
```

Comment:
aha, looks like `pending = pending + failures`
rigth?

Reply:
this is used to generate a progress bar and `pending` == `number of jobs that were not executed yet`.